src/views/Room.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Row, Col } from "react-bootstrap";
+import Peer from "peerjs";
+import useGetCamera from "../utility/useGetCamera";
+import { useNavigate } from "react-router-dom";
+import EndCall from "../components/EndCall";
+
+export default function Room() {
+    document.body.style = "background: black";
+    const id = window.location.pathname.slice(6) // pulling room id from url 
+    const peer = new Peer(id);
+    const getMedia = useGetCamera
+    const navigate = useNavigate()
+
+    React.useEffect(() => {
+        getMedia()
+
+        peer.on("call", async (call) => {
+            const userMedia = await getMedia()
+            call.answer(userMedia)
+
+            call.on("stream", async (stream) => {
+                const participant = await document.getElementById("participant")
+                participant.srcObject = stream
+
```

Comment:
This empty line shows that you are not using automatic formatting. It might be intersting for you to look into prettier.